model/Common/HistoryMeta.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace JikanPHP\Model\Common;
+
+/**
+ * Class HistoryMeta
+ *
+ * @package JikanPHP\Model
+ */
+class HistoryMeta
```

Comment:
I try to follow the models here https://github.com/jikan-me/jikan/tree/master/src/Model where does this match?
Shouldn't the mal url be updated instead?

Reply:
> I try to follow the models here https://github.com/jikan-me/jikan/tree/master/src/Model where does this match? Shouldn't the mal url be updated instead?

There is additional information included for each entry besides just the URL. Below is an example response for an individual history item.

```
    {
      "meta": {
        "mal_id": 48736,
        "type": "anime",
        "name": "Sono Bisque Doll wa Koi wo Suru",
        "url": "https://myanimelist.net/anime/48736"
      },
      "increment": 3,
      "date": "2022-01-24T05:19:28+00:00"
    }
```

Reply:
Wouldn't it be this one?
https://github.com/jikan-me/jikan/blob/master/src/Model/Common/ItemMeta.php
Can you move/rename it to match de parser source?

Reply:
I considered using `ItemMeta`, but opted for a standalone class because of it not having a `type` property and also because it expects an `imageUrl`, which the history item does not have.